### ISSUE TITLE
[A11y] CommandBar focuses disabled buttons, if it's the first button

### ISSUE THREAD

**Issue by chadsowald** [microsoft/fluentui#17421]
### Environment Information

- **Package version(s)**: @fluentui/react: ^7.148.2
- **Browser and OS versions**: Chrome latest on Win 10 Enterprise

### Describe the issue:

We use the CommandBar on many of our pages.  An accessibility bug was raised stating that when the 1st button on a command bar is disabled, it still receives TAB focus.  I am able to repro this on CodePen, so I want to ask if this is on purpose (and I should point the bug logger to the Fluent UI team) or if our product needs to fix this.  Our product could set 'allowDisabledFocus' on the button props, but again, is that the right fix?  It's unclear.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/test154/pen/JjbVdBx 

#### Actual behavior:

If the first CommandBar button is disabled, it still receives TAB focus unless 'allowDisabledFocus' is set as false.  Again, is this good/intentional?

#### Expected behavior:

It's not clear based on conflicting documentation/behavior.

### Documentation describing expected behavior

Our tester says the behavior violates the 2.1.1 a11y rules: https://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable
I see that in Fluent UI there is this comment that seems to explain why the behavior is the way it is: 
https://github.com/microsoft/fluentui/blob/d5a8c91597cd66c0e1e03ac7cb4d8ada7e0d1b30/specs/Link.md#disabled-state

I'm not sure how to reconcile these two things?  Again, does our product need to fix the bug one-off, or is this behavior by design and correct and the bug logged against our product is invalid?

**Comment by gouttierre** [comment]
@chadsowald - thank you for filing this item with us. I see the repro behavior. 
![image](https://user-images.githubusercontent.com/78806055/111645357-ea775a00-8800-11eb-8952-cbe321021c26.png)

@khmakoto -Can you confirm if this is by design where a user must add  'allowDisabledFocus' to false** for the desired functionality, or is this a regression issue?

**Comment by chadsowald** [comment]
@khmakoto @gouttierre sorry to ping, but any update here?
